frontend/assets/viaFaseProcedimentale.json

```diff
@@ -40,6 +40,20 @@
    "featureinfo":{
      "templateURL": "assets/infoTemplateFaseProcedimentale.jsxt"
    },
+  "multiLayerSelect": [
+      {
+        "name": "decsiraogc_sivia_geo:decsira_geo_sivia_linea",
```

Comment:
I think we should have the field for filtering configured here too, for all the 3 layers

Reply:
something like:

```
filterOn: "/path/attribute"
```